tests/ssl-opt.sh

```diff
@@ -63,6 +63,7 @@ guess_config_name() {
 
 O_SRV="$OPENSSL_CMD s_server -www -cert data_files/server5.crt -key data_files/server5.key"
 O_CLI="echo 'GET / HTTP/1.0' | $OPENSSL_CMD s_client"
+G_PRI="--priority=NORMAL"
```

Comment:
(commit message) Can you give a specific example of a test case that requires it?